pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -809,7 +809,7 @@ def test_fancy_getitem_slice_mixed(self, float_frame, float_string_frame):
 
         msg = r"\nA value is trying to be set on a copy of a slice from a DataFrame"
         with pytest.raises(com.SettingWithCopyError, match=msg):
-            sliced["C"] = 4.0
+            sliced.loc[:, "C"] = 4.0
```

Comment:
do the setitems (original) code behave the same (e.g. mutate & raise SettingWtihCopyError)? if so can you tests both indexers

Reply:
updated both tests